robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAndroidHttpClient.java.vm

```diff
@@ -0,0 +1,93 @@
+#if ($api >= 23)
```

Comment:
You're right sorry. I think Erich and I both got confused there.

Trevor
On Aug 21, 2015 11:56 AM, "Christian Becker" <notifications@github.com>
wrote:

> In
> robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAndroidHttpClient.java.vm
> <https://github.com/robolectric/robolectric/pull/2000#discussion_r37648494>
> :
>
> > @@ -0,0 +1,93 @@
> > +#if ($api >= 23)
>
> $api >= 23 ? Is this correct? It was removed in API 23 so maybe $api < 23
> or $api <= 22 is the corrent way?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robolectric/robolectric/pull/2000/files#r37648494>.
>